### ISSUE TITLE
Update to Accessibility page in wiki

### ISSUE THREAD

**Issue by lee0c** [allejo/jekyll-anchor-headings#39]
### Markdown Engine

Kramdown

### Snippet Usage

Not usage related but docs related.

### Expected HTML Output

n/a

### Actual HTML Output

n/a

### Additional Comments

The suggestion on your accessibility page to fully remove header permalinks for screen reader users is extremely bad practice and should be removed, especially since you have two other more workable suggestions alongside it.

If you are insistent on providing a method of removing these links, it needs to be more complete and thought-through. As it stands, using aria-hidden and tabindex will remove the element from screen reader (SR) output and prevent focus from landing on it, but change nothing visually. Partially sighted users who rely somewhat but not fully on SR, or any keyboard users, will see an element that has been styled to look focusable/like a link, but can't be reached or explained.

**Comment by lee0c** [comment]
I am happy to create the edits needed to introduce this change; please just let me know.

**Comment by allejo** [comment]
Thank you for pointing that out! I hadn't considered that approach, but you're right; another con that I hadn't considered is that there'd be bias toward abled users being able to use permalinks but not all AT users would.

I've removed the last approach from the Wiki; let me know if there's anything else that needs to be changed.
